packages/thirdweb/src/exports/react-ui.ts

```diff
@@ -0,0 +1,71 @@
+export { darkTheme, lightTheme } from "../react-ui/ui/design-system/index.js";
+export type {
+  Theme,
+  ThemeOverrides,
+} from "../react-ui/ui/design-system/index.js";
+
+export { ConnectButton } from "../react-ui/ui/ConnectWallet/ConnectWallet.js";
+export {
+  ConnectEmbed,
+  type ConnectEmbedProps,
+} from "../react-ui/ui/ConnectWallet/Modal/ConnectEmbed.js";
+
+export type {
+  ConnectButtonProps,
+  ConnectButton_connectButtonOptions,
+  ConnectButton_connectModalOptions,
+  ConnectButton_detailsButtonOptions,
+  ConnectButton_detailsModalOptions,
+} from "../react-ui/ui/ConnectWallet/ConnectWalletProps.js";
+export type { WelcomeScreen } from "../react-ui/ui/ConnectWallet/screens/types.js";
+export type { NetworkSelectorProps } from "../react-ui/ui/ConnectWallet/NetworkSelector.js";
+
+export {
+  TransactionButton,
+  type TransactionButtonProps,
+} from "../react-ui/ui/TransactionButton/index.js";
+
+export { MediaRenderer } from "../react-ui/ui/MediaRenderer/MediaRenderer.js";
+export type { MediaRendererProps } from "../react-ui/ui/MediaRenderer/types.js";
+
+// wallets
+export {
+  metamaskConfig,
+  type MetamaskConfigOptions,
+} from "../react-ui/wallets/metamask/metamaskConfig.js";
```

Comment:
these should be in /react no?

Reply:
All wallets configs hold the UI for the wallet.